peachjam/templates/peachjam/_taxonomy_tree_facet.html

```diff
@@ -2,9 +2,9 @@
 
 <div class="list-group mb-4">
   <div class="list-group-item bg-light">
-    <strong>{% trans 'Subject areas' %}</strong>
+    <strong>{{ first_level_taxonomy }}</strong>
   </div>
-  <div class="list-group-item">
+  <div class="list-group-item" style="max-height: 500px; overflow-y: auto">
```

Comment:
Elsewhere we use a `vh` max-height (eg. `25vh` on `facets-scrollable`), since the vertical size of the screen is unknown. This means we can guarantee that it never goes outside of the visible screen area (particularly on mobile). I think it would be useful to be consistent -- eg `50vh` if we think it deserves more vertical space.

I think this should be a class (`taxonomy-tree-list-group`?) rather than inline styles.